core/settings/qPref_private.h

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: GPL-2.0
+#ifndef QPREFPRIVATE_H
+#define QPREFPRIVATE_H
+
+// Header used by all qPref<class> implementations to avoid duplicating code
+
+#include <QObject>
+#include <QSettings>
+#include <QVariant>
+
+#include "core/qthelper.h"
+
+
+#define COPY_TXT(name, string)	\
+{		\
+	free((void *)prefs.name);	\
+	prefs.name = copy_qstring(string);	\
+}
```

Comment:
at least here on GitHub the trailing backslashes seem rather randomly distributed :-)

Reply:
aligned spaces or a single space should be used before the `\`, not tabs.

Reply:
Done (changes to a single space and \ not aligned)